modules/retail/pages/retail-formulas.adoc

```diff
@@ -230,14 +230,54 @@ The PXE formula is used to configure PXE booting on the branch server.
 . Navigate to the menu:Formulas[Pxe] tab, and set these parameters:
 * In the [guimenu]``Kernel filename`` field, keep the default value.
 * In the [guimenu]``Initrd filename`` field, keep the default value.
-* In the [guimenu]``Kernel commandline parameters`` field, keep the default value.
+* In the [guimenu]``Kernel command line parameters`` field, keep the default value.
+For more information about possible values, see <<retail.sect.formulas.pxe.kernelparams>>.
 * In the [guimenu]``PXE root directory`` field, enter the path to the saltboot directory (for example, [systemitem]``/srv/saltboot``).
 * In the [guimenu]``Branch id`` field, type a name to use as a branch identifier (for example: [systemitem]``Branch0001``).
 Use only alphanumeric characters for the branch identifier.
 . Click [btn]``Save Formula`` to save your configuration
 . Apply the highstate.
 
 
+
+[[retail.sect.formulas.pxe.kernelparams]]
+=== Kernel Command Line Parameters
+Saltboot supports common kernel parameters and saltboot-specific kernel parameters.
+All the parameters can be entered in the [guimenu]``Kernel Command Line Parameters`` field of the PXE formula.
+
+[systemitem]``kiwidebug=1``::
+Starts a shell on tty2 during boot and enables debug log level in Salt.
```

Comment:
Please add a warning that this should be used only in development environment and **must** not be used in production as it creates a major security hole.

Reply:
I'm going to add such a warning.